### PR TITLE
Add routes for problem count and submission clearing

### PR DESCRIPTION
**Pull request by hiliang-cmu**

### CODE REVIEW

File: picoCTF-web/api/routes/problem.py [hunk 1]
```diff
@@ -139,3 +148,11 @@ def load_problems():
 
     api.problem.load_published(data)
     return WebSuccess("Inserted {} problems.".format(len(data["problems"])))
+
+
+@blueprint.route('/clear_submissions', methods=['GET'])
+@api_wrapper
+@require_login
```
Comment (maverickwoo) [inline picoCTF-web/api/routes/problem.py]: Do we also want to `@require_admin` too? This seems to be an admin-only operation, right?

**Comment by maverickwoo** [comment]
Also, can you please add a paragraph into the commit message to explain why these routes are added? (A brief sentence or two about how these facilitates the development of graphical problem viewer would suffice.) Thanks!

File: picoCTF-web/api/routes/problem.py (hunk 1 above)
Comment (hiliang-cmu) [reply picoCTF-web/api/routes/problem.py]: Correct, added.

**Comment by hiliang-cmu** [comment]
Updated

**Pull request merged**